Doc/tools/templates/download.html

```diff
@@ -13,7 +13,7 @@ <h1>Download Python {{ release }} Documentation</h1>
 
 <p>To download an archive containing all the documents for this version of
 Python in one of various formats, follow one of links in this table. The numbers
-in the table are the size of the download files in megabytes.</p>
+in the table are the size of the download files in mebibytes.</p>
```

Comment:
it's not a commonly used unit (I mean, everybody misuses "megabytes", but "mebibytes" term is rarely used), maybe add its value: megabytes (2\ :sup:`20` bytes).

I didn't test if it's formatted properly, :superscript: is for the 2^20 exponent.

Reply:
Alright :) I translated the idea to HTML.